experimental/cloudstrapper/playbooks/roles/aws-cleanup/tasks/keys.yaml

```diff
@@ -0,0 +1,11 @@
+---
+
+- name: Delete KeyBoot
+  ec2_key:
+    name: "{{ keyBoot }}"
+    state: absent
+
+- name: Delete KeyHost
+  ec2_key:
+    name: "{{ keyHost }}"
+    state: absent
```

Comment:
It is possible customers use their pre-existing keys and not the ones that were generated as part of Cloudstrapper. In that case, how can we protect customers from accidentally deleting these keys? If there is a way to ensure that these keys were created as part of CLoudstrapper and not inherited from elsewhere, that would be ideal. Maybe a tag added in keyCreate which can be verified here before deleting?

Reply:
I added tagging of keys and I am checking the tags before deleting them!